src/dotnet-test-console/project.json

```diff
@@ -0,0 +1,38 @@
+﻿{
+  "version": "1.0.0-featmsbuild-*",
+  "buildOptions": {
+  	"keyFile": "../../tools/Key.snk",
+    "compile": {
+      "include": [
+        "**/*.cs",
+        "../dotnet/CommandLine/*.cs"
+      ]
+    }
+  },
+
+  "dependencies": {
+    "System.Diagnostics.TraceSource": "4.0.0",
+    "System.Diagnostics.TextWriterTraceListener": "4.0.0",
+    "Microsoft.DotNet.Cli.Utils": {
+      "target": "project"
+    },
+    "Microsoft.DotNet.ProjectModel": {
+      "target": "project"
+    },
+    "Microsoft.Extensions.Testing.Abstractions": {
+      "target": "project"
+    },
+    "Microsoft.DotNet.InternalAbstractions": {
+  	  "target": "project"
+  	},
+    "Microsoft.DotNet.PlatformAbstractions": "1.0.1-beta-000919"
+  },
+
+  "frameworks": {
+    "netstandard1.6": {
```

Comment:
If it is a library, I think it should be named like a library.

---
In reply to: [76426587](https://github.com/dotnet/cli/pull/4102#discussion_r76426587) [](ancestors = 76426587)